.github/workflows/website.yml

```
@@ -0,0 +1,26 @@
+name: website
+on:
+  push:
+    branches:
+      - master
+permissions:
+  contents: none
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  FASTLY_API_TOKEN: ${{ secrets.FASTLY_API_TOKEN }}
+jobs:
+  build-website:
+    runs-on: ubuntu-latest
```

Instead of needing to `cd website` in each command, you should be able to do this:
```suggestion
    runs-on: ubuntu-latest
        defaults:
            run:
                working-directory: website
```